### DISCUSSION TITLE
How to stop teleport agent/service

### DISCUSSION THREAD

**Discussion by icanche** [gravitational/teleport#8358] [Q&A] [answered]
Hello, i would like to know if there is a way to stop the teleport service, i have tried service teleport stop but the service doesnt recognize that command, is there a way to stop it without kill ?

**Comment by stevenGravy** [comment, marked as answer]
@icanche please provide how you configured it.  If this was a systemd on a ubuntu system then `sudo service teleport stop` should work.

**Comment by icanche** [comment]
Hello i was just doing teleport start with nohup, now i configured with systemd and it works as you said. Thank you!

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Steven Martin ***@***.***>
Enviado: Friday, September 24, 2021 10:31:32 PM
Para: gravitational/teleport ***@***.***>
Cc: Jorge Imar Canché Álvarez ***@***.***>; Mention ***@***.***>
Asunto: Re: [gravitational/teleport] How to stop teleport agent/service (Discussion #8358)

@icanche please provide how you configured it. If this was a systemd on a ubuntu system then sudo service teleport stop should work. — You are receiving this because you were mentioned. Reply to this
¡Advertencia! Este mensaje se envió desde fuera de su organización.     Allow sender<https://mail-cloudstation-us-west-2.prod.hydra.sophos.com/mail/api/xgemail/smart-banner/8ce3bb2e786740cb1e0211a5c72369ae> | Block sender<https://mail-cloudstation-us-west-2.prod.hydra.sophos.com/mail/api/xgemail/smart-banner/7cf79aea4c37776104ebc598a7c54a72>
sophospsmartbannerend

@icanche<https://us-west-2.protection.sophos.com?d=github.com&u=aHR0cHM6Ly9naXRodWIuY29tL2ljYW5jaGU=&i=NTg3ZmUyYzFlMDlhMGExMGFkZGUwZGVm&t=cHlMUHlkbHpsVUxyOE5vNWFoWFBNTHppR05Sb1dNVkxjaXFDSVJHNExnVT0=&h=7be224a9d40b41169e11ee12402f1975> please provide how you configured it. If this was a systemd on a ubuntu system then sudo service teleport stop should work.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://us-west-2.protection.sophos.com?d=github.com&u=aHR0cHM6Ly9naXRodWIuY29tL2dyYXZpdGF0aW9uYWwvdGVsZXBvcnQvZGlzY3Vzc2lvbnMvODM1OCNkaXNjdXNzaW9uY29tbWVudC0xMzgyNTk5&i=NTg3ZmUyYzFlMDlhMGExMGFkZGUwZGVm&t=RVRGVVVzZ0loaEFLWk1EeW8yMENobEV0Z2ZlY29LNkJuL1FVSEplWWlzUT0=&h=7be224a9d40b41169e11ee12402f1975>, or unsubscribe<https://us-west-2.protection.sophos.com?d=github.com&u=aHR0cHM6Ly9naXRodWIuY29tL25vdGlmaWNhdGlvbnMvdW5zdWJzY3JpYmUtYXV0aC9BVllPS0Q1NlZXUERCQlJFN01JTU01RFVEVTdCSEFOQ05GU001RVVKUUVEQQ==&i=NTg3ZmUyYzFlMDlhMGExMGFkZGUwZGVm&t=cWRBL0NpRk85VUVBcC8rMUJoTzAyRjdRRU9Kb3gwSTRoVVZrYzdUWFZFcz0=&h=7be224a9d40b41169e11ee12402f1975>.
Triage notifications on the go with GitHub Mobile for iOS<https://us-west-2.protection.sophos.com?d=apple.com&u=aHR0cHM6Ly9hcHBzLmFwcGxlLmNvbS9hcHAvYXBwbGUtc3RvcmUvaWQxNDc3Mzc2OTA1P2N0PW5vdGlmaWNhdGlvbi1lbWFpbCZtdD04JnB0PTUyNDY3NQ==&i=NTg3ZmUyYzFlMDlhMGExMGFkZGUwZGVm&t=ODJBVFBoMVVsdnhWdVF2ajJtdFlWWDU3bXMzVWhUcUhqNjYzOWJHT0YyST0=&h=7be224a9d40b41169e11ee12402f1975> or Android<https://us-west-2.protection.sophos.com?d=google.com&u=aHR0cHM6Ly9wbGF5Lmdvb2dsZS5jb20vc3RvcmUvYXBwcy9kZXRhaWxzP2lkPWNvbS5naXRodWIuYW5kcm9pZCZyZWZlcnJlcj11dG1fY2FtcGFpZ24lM0Rub3RpZmljYXRpb24tZW1haWwlMjZ1dG1fbWVkaXVtJTNEZW1haWwlMjZ1dG1fc291cmNlJTNEZ2l0aHVi&i=NTg3ZmUyYzFlMDlhMGExMGFkZGUwZGVm&t=NW9KdSs0NjhubGNqc0xNVUVTRWRYOTk0bUc4a2N2ZE0yQ2RCS21zQUNJRT0=&h=7be224a9d40b41169e11ee12402f1975>.
